### PR TITLE
TextInput / InputGroup: prominent variant

### PR DESCRIPTION
**Pull request by chrisjbrown**
**This change:** (check at least one)

- [x] Adds a new feature
- [ ] Fixes a bug
- [ ] Improves maintainability
- [ ] Improves documentation
- [ ] Is a release activity

**Is this a breaking change?** (check one)

- [ ] Yes
- [x] No

**Is the:** (complete all)

- [ ] Title of this pull request clear, concise, and indicative of the issue number it addresses, if any?
- [ ] Test suite(s) passing?
- [ ] Code coverage maximal?
- [ ] Changeset added?

**What does this change address?**
A clear and concise description or a direct link to an issue [e.g. #15]
If adding a feature, a rationale for its addition goes here.

**How does this change work?**
A clear, detailed description of how this change addresses the issue.
This could be a bullet list if there are several moving parts.

**Additional context**
Add any other context here.

### CODE REVIEW

File: .changeset/neat-shirts-doubt.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@ithaka/pharos": patch
+---
+
+feat: cozy(name wip) text input variant
```
Comment (daneah) [inline .changeset/neat-shirts-doubt.md:5]:
```suggestion
```

File: packages/pharos/src/components/input-group/pharos-input-group.ts [hunk 2]
```diff
@@ -41,7 +45,7 @@ export class PharosInputGroup extends PharosTextInput {
   protected override updated(changedProperties: PropertyValues): void {
     super.updated(changedProperties);
 
-    if (changedProperties.has('_prependGroupWidth')) {
+    if (changedProperties.has('_prependGroupWidth') && this.variant !== 'cozy') {
```
Comment (daneah) [inline packages/pharos/src/components/input-group/pharos-input-group.ts:48]: This logic is kind of murky—not sure I'd understand why it's here, out of context. Any way we can clarify that?

File: packages/pharos/src/utils/scss/_mixins.scss [hunk 3]
```diff
@@ -83,6 +83,13 @@
     calc(var(--pharos-spacing-three-quarters-x) - 1px);
 }
 
+/// Styles for input states with thicker borders
```
Comment (daneah) [inline packages/pharos/src/utils/scss/_mixins.scss:86]:
```suggestion
/// Styles for cozy input states with thicker borders
```

File: packages/pharos/src/components/input-group/pharos-input-group.ts (hunk 2 above)
Comment (chrisjbrown) [reply packages/pharos/src/components/input-group/pharos-input-group.ts]: made an attempt. not sure it's much of an improvement 🤔

Comment (daneah) [reply packages/pharos/src/components/input-group/pharos-input-group.ts]: Agreed—I guess I'd ask: Why does the prepend not get used for this variant?

File: packages/pharos/src/components/input-group/pharos-input-group.scss [hunk 4]
```diff
@@ -37,3 +37,21 @@ slot[name='prepend']::slotted([data-pharos-component='PharosIcon']:first-child)
...
+  }
+
+  slot:not([name])::slotted(:last-child) {
+    margin-right: var(--pharos-spacing-1-x);
+  }
+
+  slot:not([name])::slotted([data-pharos-component='PharosIcon']:last-child) {
+    margin-right: var(--pharos-spacing-1-x);
+  }
+}
```
Comment (daneah) [inline packages/pharos/src/components/input-group/pharos-input-group.scss:57]: I think these should all be `--pharos-spacing-three-quarters-x`

Comment (chrisjbrown) [reply packages/pharos/src/components/input-group/pharos-input-group.scss:57]: i thought based on the conversion we want to be uniformally 1-x everywhere?

File: packages/pharos/src/components/input-group/pharos-input-group.ts (hunk 2 above)
Comment (chrisjbrown) [reply packages/pharos/src/components/input-group/pharos-input-group.ts]: actually i think it should be used 🤔 
but will use a different value to make it's adjustment... I've made an update
and created more examples in storybook with the prepend slot used

File: packages/pharos/src/components/input-group/pharos-input-group.scss (hunk 4 above)
Comment (chrisjbrown) [reply packages/pharos/src/components/input-group/pharos-input-group.scss]: updated

**Review by jialin-he — APPROVED** [review]

File: packages/pharos/src/components/input-group/pharos-input-group.ts (hunk 2 above)
Comment (daneah) [reply packages/pharos/src/components/input-group/pharos-input-group.ts]: The stories were def helpful 👍🏻

Comment (chrisjbrown) [reply packages/pharos/src/components/input-group/pharos-input-group.ts]: @daneah let me know if you have anything else for this

Comment (daneah) [reply packages/pharos/src/components/input-group/pharos-input-group.ts]: I think this looks great now.

**Review by daneah — APPROVED** [review]

**Pull request merged**